packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -36,15 +36,15 @@ export const FinalFormSelect = <T,>({
         }
     },
     children,
-    clearable,
+    required,
```

Comment:
Was `required` previously included in `...rest`? 
Then it needs to be added to `selectProps` so it's still passed to the `<Select />`.

Reply:
Yes, it was missing. It's added now